src/Underworld.ts

```diff
@@ -1465,24 +1463,49 @@ export default class Underworld {
       console.error('Could not find biome for levelIndex: ', levelIndex);
     }
 
-    const isTutorialRun = !isTutorialComplete();
+    // isTutorialRun will always be false on headless
+    // If tutorial isn't complete, make this a tutorial run
     if (levelIndex == 0) {
-      isFirstEverPlaySession = !isFirstTutorialStepComplete();
-      if (isFirstEverPlaySession) {
+      this.isTutorialRun = !isTutorialComplete();
+      if (this.isTutorialRun) {
         console.log('Set gamemode to "tutorial" so that the first playthrough is easier');
         this.gameMode = 'tutorial';
+        if (!isTutorialFirstStepsComplete(['portal'])) {
+          // Set the level index to -1 to spawn the first tutorial level
+          levelIndex = -1;
+
+          // Give the player default tutorial cards
+          if (globalThis.player) {
+            for (let spell of [slashCardId, arrowCardId, pushId]) {
+              const upgrade = Upgrade.getUpgradeByTitle(spell);
+              if (upgrade) {
+                this.forceUpgrade(globalThis.player, upgrade, false);
+              }
+            }
```

Comment:
I love this change! Well done

Reply:
@jdoleary 
Awesome! I didn't put too much thought into which spells should be default. Can you think of a better tutorial loadout, or do you think this is fine?

Reply:
How about Target Cone instead of Arrow to give them a feel for the targeting mechanics?

Reply:
I like that! Will do.